code/game/objects/items/devices/gps.dm

```diff
@@ -15,6 +15,9 @@
 	. = ..()
 	AddComponent(/datum/component/gps/item, gpstag)
 
+/obj/item/gps/spaceruin
+	gpstag = SPACE_SIGNAL_GPSTAG
```

Comment:
If you're going to keep unfixed GPS signals, can you at least randomize them so that people don't accidentally visit the same one twice if they're confused? Or can communicate about it easier? As in, so I can say "Let's try the alpha distress signal" or whatever instead of "the distress signal at X 126 and Y 62".

Reply:
Or alternatively, just toss on a prefix variable that can be adjusted in the map files themselves and can be var-edited at the creator's discretion.

Reply:
Go out into the ruins. Track down their GPS. Rename them as necessary.

You can tell which signal is which by the fact they have a set of coordinates attached to them that are static unless someone moves the GPS.

If that is too high a barrier of entry for players to be able to explore either alone or in a group, then I don't know what to say other than... Learn basic numeracy skills?

I am discouraging snowflake GPS signals in ruins entirely. They're gone. They're generic. You should navigate by coordinates. That is intended behaviour.